### PR TITLE
Add docs and jetson4.6 instructions

### PR DESCRIPTION
**Pull request by goncalossantos**
Adds support for Jetson 4.6
Slight improvements on Quickstart/Intro docs
Fixes Jetson start scripts but with shared volumes
Improves inference.py resilience with defaultdict

### CODE REVIEW

**Comment by goncalossantos** [comment]
I wonder if we shouldn't just merge both Jetson directories as they are exactly the same, with the exception that the jetson46 dir doesn't need the tensorflow image

**Comment by jaypatelh** [comment]
Thanks for this PR @goncalossantos! Looks good.

**Review by jaypatelh — APPROVED** [review]

**Comment by jaypatelh** [comment]
Gentle ping on this @goncalossantos. See my latest comment above.

File: start_jetson_container.sh [hunk 1]
```diff
@@ -12,4 +12,4 @@ if [ -z "${t}" ]
     usage
 fi
 
-sudo docker run -v /usr:/usr -v /dev/shm/aip/images:/dev/shm/aip/images --rm --device=/dev/nvhost-ctrl  --device=/dev/nvhost-ctrl-gpu  --device=/dev/nvhost-prof-gpu  --device=/dev/nvmap  --device=/dev/nvhost-gpu  --device=/dev/nvhost-as-gpu --runtime=nvidia --network=host -it "${t}"
```
[the remark this answers is not in the document]
Comment (jaypatelh) [reply start_jetson_container.sh:15]: Added the `/usr` mount myself

**Pull request merged**